app/views/users/_profile_card.html.erb

```diff
@@ -17,7 +17,14 @@
             <div class="learning-goal-wrapper">
               <p class="camel bold learning-goal-title">Learning Goal</p>
               <p class="learning-goal">
-                <%= user.learning_goal || 'Not Specified' %>
+                <%= user.learning_goal || ''  %>
```

Comment:
I think we should do this a bit differently. if the user.learning goal isn't set it will return nil so it will use the value at the other side of the or pipes `||`

so we could do something like this:
```
<%= user.learning_goal || link_to "Settings", edit_user_registration_path, class: 'accent'  %>
```

But I think we should include a bit more than just "settings" if the user doesn't have a learning goal set.
maybe something like "set a learning goal in your settings"

The solution to this imo is to create a helper method in the user_helper.rb file:
```
def set_learning_goal
  <%= link_to "set a learning goal in your settings", edit_user_registration_path, class: 'accent' %>
end
```

and then use it in the view like so:
```
<%= user.learning_goal || set_learning_goal  %>
```